scan-build-wrapper.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+scan-build -v --status-bugs -disable-checker unix.Malloc "$@"
```

Comment:
Why do we want to disable `unix.Malloc`?

Reply:
because scan-build does not understand auto-cleanup. This has been disabled before, too (in the workflow file).